apps/shell/app.js

```diff
@@ -58,9 +58,9 @@ app.use(process.env.PASSENGER_BASE_URI || '/', router);
 const server = new http.createServer(app);
 const wss = new WebSocket.Server({ noServer: true });
 
-let host_whitelist = new Set;
-if (process.env.SSHHOST_WHITELIST){
-  host_whitelist = new Set(process.env.SSHHOST_WHITELIST.split(':'));
+let host_allowlist = new Set;
+if (process.env.SSHHOST_ALLOWLIST){
+  host_allowlist = new Set(process.env.SSHHOST_ALLOWLIST.split(':'));
```

Comment:
I don't know if we've standardized this or talked about it, but it seems like we should always pre-pend OOD to environment variables? 

So this would turn into `OOD_SSHHOST_ALLOWLIST` to avoid collisions, but also just 'look and feel like' other OOD environment variable configurations.  I feel like this is especially dangerous with `SSH*`.